src/main/groovy/com/getkeepsafe/dexcount/sdkresolver/SdkResolver.groovy

```diff
@@ -99,26 +115,32 @@ class SdkResolver {
         if (userAndroid.exists()) {
             log.debug "Found existing SDK at '$userAndroid.absolutePath'. Writing to $FN_LOCAL_PROPERTIES."
 
-            writeLocalProperties userAndroid.absolutePath
+            writeLocalProperties(userAndroid.absolutePath, ndkDir)
```

Comment:
@benjamin-bader was right.  On line 89 you have: `ndkDir = new File(androidNdkHome)`, so it is a file.  This causes the tests to fail since it can't compile that.  I've got a PR incoming.